lib/system-nixpkgs-map.nix

```diff
@@ -17,8 +17,12 @@ pkgs:
      GL = pkgs.libGL;
      GLU = pkgs.libGLU;
      alut = pkgs.freealut;
-     X11 = pkgs.xorg.libX11;
+     X11 = pkgs.buildEnv {
```

Comment:
This could work, but have a look at `lib/pkgconf-nixpkgs-map.nix`. There is a list of packages for each name. We should do the same here.

Reply:
good catch! this should be fixed now.